src/reaktoro_pse/tutorials/integration_with_ro.ipynb

```diff
@@ -1838,7 +1784,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "reaktoro-pse-dev",
+   "display_name": "watertap-dev",
```

Comment:
This change should probably be reverted, i.e. we should keep the kernel name as `reaktoro-pse-dev` for consistency.

Reply:
I was useing my Watertap-dev env to run this, rerun them with reaktoro-pse-dev